recipes-aos/aos-iamanager/aos-iamanager_git.bb

```diff
@@ -21,8 +21,16 @@ SRC_URI += " \
 
 DEPENDS += "poco systemd grpc grpc-native protobuf-native protobuf openssl curl libnl"
 
-OECMAKE_GENERATOR = "Unix Makefiles"
-EXTRA_OECMAKE += "-DFETCHCONTENT_FULLY_DISCONNECTED=OFF -DWITH_MBEDTLS=OFF -DWITH_OPENSSL=ON"
+EXTRA_OECMAKE += " \
```

Comment:
Can we add packages config? https://docs.yoctoproject.org/2.4.4/ref-manual/ref-manual.html#var-PACKAGECONFIG

Reply:
Done